plugins/kubernetes_ng/app/javascript/widgets/app/routes/clusters/-components/ClusterList.tsx

```diff
@@ -50,7 +52,7 @@ const ClusterList: React.FC<ClusterListProps> = ({ clusters = [], error, isLoadi
       return (
         <DataGridRow>
           <DataGridCell colSpan={NUMBER_OF_COLUMNS}>
-            <InlineError error={error} />
+            <InlineError error={error} data-testid="clusters-list-error-state" />
```

Comment:
here as well, I think we can grab error by its text `data-testid` might not be needed.